Automattic/jetpack
# Rnmobile/fix/tiled gallery

#### Changes proposed in this Pull Request:
* The project's Phase 1B is scoped to only include the Tiled Gallery's Square layout. This PR removes code associated with the block's other layouts.

#### Jetpack product discussion
p9ugOq-1Tb-p2

#### Does this pull request change what data or activity we track or use?
No, it does not.

#### Testing instructions:

Verify that the Square layout works on mobile and that other layouts are not available.

<details>
<summary>Setup to test from the main apps</summary>
<p>
<ol>
<li>Run the packager from the <code>add/tiled-gallery-block</code> branch: <code>npm run start:reset</code></li>
<li>Run either of the main apps (WPiOS or WPAndroid)</li>
<li>Open a new post in the main apps and expect the editor to load from the running packager</li>
</details>

1. Launch the iOS app
2. Add the Tiled Gallery block
3. Add images to it
4. Preview the post 
5. Expect the layout to match that of the web's Tiled Gallery Square layout
6. Repeat steps 1 to 5 on the Android app
